src/AggregateFunctions/AggregateFunctionGroupBitmapData.h

```diff
@@ -122,7 +122,7 @@ class RoaringBitmapWithSmallSet : private boost::noncopyable
             size_t size;
             readVarUInt(size, in);
 
-            static constexpr size_t max_size = 1_GiB;
+            static constexpr size_t max_size = 100_GiB;
```

Comment:
Having such a high limit here seems dangerous. It seems like it will allocate a huge amount of memory without checking max_memory_usage, leading to a server being killed by OOM.

I've tested it doing something like:
```
static constexpr size_t max_size = 1000_GiB;
```

```sql
SELECT CAST(unhex('0181808080908380000000'), 'AggregateFunction(groupBitmap, UInt64)')
SETTINGS max_memory_usage = 536870912
```

```
jun 06 13:34:14 Mordor clickhouse[338865]: 2023.06.06 11:34:14.015792 [ 339142 ] {} <Trace> AsynchronousMetrics: MemoryTracking: was 87.27 GiB, peak 87.27 GiB, free memory in arenas 596.00 KiB, will set to 89.86 GiB (RSS), difference: 2.60 GiB
jun 06 13:34:14 Mordor clickhouse[338865]: 2023.06.06 11:34:14.017429 [ 339142 ] {} <Debug> MemoryTracker: Current memory usage (total): 89.86 GiB.
jun 06 13:34:15 Mordor clickhouse[338819]: 2023.06.06 11:34:15.129934 [ 338819 ] {} <Fatal> Application: Child process was terminated by signal 9 (KILL). If it is not done by 'forcestop' command or manually, the possible cause is OOM Killer (see 'dmesg' and look at the '/var/log/kern.log' for the details).
```